app/controllers/frontend_log_controller.rb

```diff
@@ -46,6 +46,12 @@ class FrontendLogController < ApplicationController
   }.freeze
   # rubocop:enable Layout/LineLength
 
+  ALLOWED_EVENTS = [
+    :phone_input_country_changed,
+  ].freeze
```

Comment:
unless we're planing to mix strings and symbols we can do this:
```suggestion
  ALLOWED_EVENTS = %i[
    phone_input_country_changed
  ].freeze
```

Reply:
Yep, good idea!